python/demo/poisson/demo_poisson.py

```diff
@@ -197,6 +196,7 @@
 # for later visualization and also plot it using
 # the :py:func:`plot <dolfinx.common.plot.plot>` command: ::
 
+u.vector.ghostUpdate(addv=PETSc.InsertMode.INSERT, mode=PETSc.ScatterMode.FORWARD)
```

Comment:
Is ghostUpdate needed after solve(a == L)?

Reply:
It's needed for correct visualisation on the boundaries with `dolfinx.plotting.plot(u)`